.gitignore

```diff
@@ -4,6 +4,8 @@
 .vscode/
 *.code-workspace
 inspect.lua
+src/TreeData/*.png
+*.png
```

Comment:
Remove this change

Reply:
reverted back to the original .gitignore, still showing a change based off of most recent pull
![image](https://github.com/PathOfBuildingCommunity/PathOfBuilding/assets/70865828/b1d3378c-6c01-440c-ae8e-3fae1a42a8d6)